FirebaseInstallations/Source/Library/InstallationsStore/FIRInstallationsStoredAuthToken.h

```diff
@@ -14,26 +14,45 @@
  * limitations under the License.
  */
 
-// TODO: Add short docs to the API
+// TODO: Add short docs to the undocumented API.
 #import <Foundation/Foundation.h>
 
 NS_ASSUME_NONNULL_BEGIN
 
+/**
+ *  The enum represent possible states of the installation auth token.
+ *
+ * WARNING: The enum is stored to Keychain as a part of `FIRInstallationsStoredAuthToken`.
+ * Modification of it can lead to incompatibility with previous version. Any modification must be
+ * evaluated and, if it is really needed, the `storageVersion` must be bumped and proper migration
+ * code added.
+ */
 typedef NS_ENUM(NSInteger, FIRInstallationsAuthTokenStatus) {
-  // An initial status or an undefined value.
+  /// An initial status or an undefined value.
   FIRInstallationsAuthTokenStatusUnknown,
+  /// The auth token has been requested from the server but the response has not been received yet.
   FIRInstallationsAuthTokenRequestInProgress,
+  /// The auth token has been received from the server.
   FIRInstallationsAuthTokenStatusTokenReceived
 };
 
+/**
+ * The class is be used by `FIRInstallationsStore`. It is required to
```

Comment:
I think starting out with "This class serializes and deserializes the installation data into/from `NSData` ..." followed by "This class is primarily used by `FIRInstallationsStore`. Better to describe what it is first then what it's used for instead of the other way around.